libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx

```diff
@@ -38,11 +55,21 @@ export const TemplatesDesigner = ({
   return (
     <>
       <TemplatePanel onCreateClick={onCreateClick} />
-      <div className="msla-templates-list">
-        {filteredTemplateNames?.map((templateName: string) => (
-          <TemplateCard key={templateName} templateName={templateName} />
-        ))}
-      </div>
+      {filteredTemplateNames && filteredTemplateNames?.length > 0 ? (
+        <div className="msla-templates-list">
+          {filteredTemplateNames.map((templateName: string) => (
+            <TemplateCard key={templateName} templateName={templateName} />
+          ))}
+        </div>
+      ) : (
+        <div className="msla-templates-empty-list">
+          <EmptySearch />
```

Comment:
NIT: I am just wondering if we can put some alt text or move it to an image tag just to make sure for whenever it doesn't load fine or screen readers.

Reply:
Since it's a svg which is just a html being imported as a react component, I don't see a place to add alt text. Through research, it seems like I can put aria-label, so I will do that to the <div> in svg!

Reply:
Since the image is presentational only and not giving real information (empty search results should be announced on search update, make sure that happens) you can mark it as display presentation in CSS and screen readers will just skip over it. No need for alt text on this one I don’t think.